### ISSUE TITLE
error with composed repositories

### ISSUE THREAD

**Issue by iilyak** [ffizer/ffizer#409]
The following `.ffizer.yaml` produces following error
```
variables:
  - name: author_name
    default_value: '{{ env_var "USERNAME" }}'

ignores:
  - .git # exclude .git of the template host

imports:
  - uri: "https://github.com/ffizer/templates_default.git"
    rev: "master"
    subfolder: "oss_license"
```

```
ffizer test-samples --source .
Jul 15 11:58:36.085 ERROR ffizer: cmd: CliOpts {
    verbose: 0,
    cmd: TestSamples(
        TestSamplesOpts {
            src: SourceLoc {
                uri: SourceUri {
                    raw: ".",
                    path: ".",
                    host: None,
                },
                rev: "master",
                subfolder: None,
            },
            offline: false,
        },
    ),
}
Jul 15 11:58:36.086 ERROR ffizer: failed: SerdeYaml {
    source: Scan(
        ScanError {
            mark: Marker {
                index: 30,
                line: 1,
                col: 30,
            },
            info: "while scanning a tag, did not find expected whitespace or line break",
        },
    ),
}
```

If I comment out imports it works fine.

The error message is very cryptic and it is hard to guess that is caused by `imports` section.

**Comment by iilyak** [comment]
I tried it on master and `2.3.0`

**Comment by iilyak** [comment]
The online yaml validation tool confirms that the yaml file is syntactically correct

**Comment by davidB** [comment]
The error seems to be on the imported template.
I'll try
- to update and to fix the template
- to add CI on the template(s),
- to improve the reporting of the error

Thanks for reporting, sorry for the bug.

**Comment by davidB** [comment]
To fix I needed to release ffizer 2.4.1 (you can run `ffizer upgrade` to update your local binary) and to update the oss_license template.

Feel free to reopen or to comment.

**Comment by iilyak** [comment]
Thank you for a quick fix
